Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -195,6 +195,8 @@ public void ClearRenderers()
                     RemoveRenderer(renderers.Count - 1);
                 }
             }
+
+            UseOnPreRender = false;
```

Comment:
Interesting. Does this mean `OnDisable` isn't getting called when the monobehaviour is destroyed?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/a69eceee3e248b41f48f2e036bdce92839c933b6/Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs#L234-L247

Reply:
I also feel like this approach _could_ create a CameraEventRouter at shutdown just to remove a not-already-registered event. 

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/a69eceee3e248b41f48f2e036bdce92839c933b6/Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs#L77-L98

maybe we should also check that `useOnPreRender != value`? Or only set `UseOnPreRender` to false here if it's true?

Reply:
Oh yeah I guess OnDisable != OnDestroy? Or maybe it's a specific scenario where when you delete it in editor rather than via the Destroy() call that things get weird.

Reply:
I'll take another pass to ensure we aren't messing anything up here

Reply:
Turns out the event was getting added to the handler multiple times. Doing a useOnPreRenderer != value check to the setter alleviated this issue.